cmd/lotus-storage-miner/init.go

```diff
@@ -105,29 +102,34 @@ var initCmd = &cli.Command{
 			return err
 		}
 
-		if err := storageMinerInit(ctx, cctx, api, r); err != nil {
-			fmt.Printf("ERROR: failed to initialize lotus-storage-miner: %s\n", err)
-			fmt.Println("Cleaning up after attempt...")
-			if err := os.RemoveAll(repoPath); err != nil {
-				fmt.Println("ERROR: failed to clean up failed storage repo: ", err)
+		lr, err := r.Lock(repo.StorageMiner)
+		if err != nil {
+			return err
+		}
+
+		if err := storageMinerInit(ctx, cctx, api, lr); err != nil {
+			log.Errorf("Failed to initialize lotus-storage-miner: %s", err)
```

Comment:
This could be
```suggestion
			log.Errorf("Failed to initialize lotus-storage-miner: %+v", err)
```

To print more useful errors